pocs/utils/images/__init__.py

```diff
@@ -90,7 +90,20 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
     if fname.endswith('.cr2'):
         return _make_pretty_from_cr2(fname, timeout=timeout, **kwargs)
     elif fname.endswith('.fits'):
-        return _make_pretty_from_fits(fname, **kwargs)
+        pretty_path = _make_pretty_from_fits(fname, **kwargs)
+
+        # Symlink latest.jpg to the image; first remove the symlink if it already exists.
+        latest_path = '{}/images/latest.jpg'.format(os.getenv('PANDIR'))
```

Comment:
What do you think about moving the linking for CR2 files from the cr2_to_jpg.sh script into this file?
Then we don't have two different approaches. And we don't have the primary==False check in just the CR2 case, and not the FITS case?

Reply:
I would almost prefer to go the other way and bury the details of the linking inside `_make_pretty_from_fits`, which is almost what I did in the first place. 

For the CR2 we use the `convert` shell command. We could split this out of the existing bash script but then would just be calling another bash script. Not a huge deal either way.